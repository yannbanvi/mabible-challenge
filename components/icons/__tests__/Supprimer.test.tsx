/** @jest-environment jsdom */
import '@testing-library/jest-dom'
import { render } from '@testing-library/react'
import SupprimerIcon from '../Supprimer'
describe('SupprimerIcon', () => {
  it('renders SupprimerIcon page successfully', () => {
    const { container } = render(<SupprimerIcon />)
    const svg = container.getElementsByTagName('svg');
    expect(svg).toHaveLength(1);
    expect(container).toMatchSnapshot();
  })
});