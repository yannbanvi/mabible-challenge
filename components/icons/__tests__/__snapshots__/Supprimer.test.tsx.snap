// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SupprimerIcon renders SupprimerIcon page successfully 1`] = `
<div>
  <svg
    fill="none"
    height="18"
    viewBox="0 0 18 18"
    width="18"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M6.11774 5.66042H6.9411V14.8387H6.11774V5.66042Z"
      fill="#211B28"
    />
    <path
      clip-rule="evenodd"
      d="M5.87881 5.66042C5.87881 5.52845 5.98578 5.42148 6.11774 5.42148H6.9411C7.07307 5.42148 7.18004 5.52845 7.18004 5.66042V14.8387C7.18004 14.9707 7.07307 15.0776 6.9411 15.0776H6.11774C5.98578 15.0776 5.87881 14.9707 5.87881 14.8387V5.66042ZM6.11774 5.66042V14.8387H6.9411V5.66042H6.11774Z"
      fill="#211B28"
      fill-rule="evenodd"
    />
    <path
      d="M11.0583 5.66042H11.8817V14.8387H11.0583V5.66042Z"
      fill="#211B28"
    />
    <path
      clip-rule="evenodd"
      d="M10.8194 5.66042C10.8194 5.52845 10.9264 5.42148 11.0583 5.42148H11.8817C12.0136 5.42148 12.1206 5.52845 12.1206 5.66042V14.8387C12.1206 14.9707 12.0136 15.0776 11.8817 15.0776H11.0583C10.9264 15.0776 10.8194 14.9707 10.8194 14.8387V5.66042ZM11.0583 5.66042V14.8387H11.8817V5.66042H11.0583Z"
      fill="#211B28"
      fill-rule="evenodd"
    />
    <path
      d="M16.4104 2.74212H12.7049L12.705 1.49059C12.705 0.823103 12.1287 0.238938 11.4698 0.238938H6.52927C5.87056 0.238938 5.29408 0.82307 5.29408 1.49059V2.74225L1.58891 2.74212V3.57646H2.8241L3.5652 16.5929C3.5652 17.2604 4.14151 17.7611 4.80038 17.7611H13.1168C13.7755 17.7611 14.352 17.2604 14.352 16.5929L15.0931 3.57646L16.4103 3.57659L16.4104 2.74212ZM6.1178 1.49059C6.1178 1.24027 6.28245 1.07341 6.5295 1.07341H11.47C11.7171 1.07341 11.8817 1.24025 11.8817 1.49059V2.74225H6.11793L6.1178 1.49059ZM13.611 16.5095C13.611 16.7599 13.4463 16.9267 13.1993 16.9267H4.80035C4.55332 16.9267 4.38866 16.7599 4.38866 16.5095L3.64756 3.57668H14.2694L13.611 16.5095Z"
      fill="#211B28"
    />
    <path
      clip-rule="evenodd"
      d="M16.6493 2.50318L16.6492 3.81555L15.3188 3.81542L14.5909 16.6C14.5869 17.4139 13.8864 18 13.1168 18H4.80038C4.03062 18 3.33033 17.4138 3.32628 16.6L2.59838 3.8154H1.34998V2.50317L5.05514 2.5033V1.49059C5.05514 0.694098 5.73563 0 6.52927 0H11.4698C12.2636 0 12.9439 0.694155 12.9439 1.49059L12.9438 2.50318H16.6493ZM12.7049 2.74212H16.4104L16.4103 3.57659L15.0931 3.57646L14.352 16.5929C14.352 17.2604 13.7755 17.7611 13.1168 17.7611H4.80038C4.14151 17.7611 3.5652 17.2604 3.5652 16.5929L2.8241 3.57646H1.58891V2.74212L5.29408 2.74225V1.49059C5.29408 0.82307 5.87056 0.238938 6.52927 0.238938H11.4698C12.1287 0.238938 12.705 0.823103 12.705 1.49059L12.7049 2.74212ZM13.372 16.5035L14.018 3.81562H3.90058L4.6276 16.5027V16.5095C4.6276 16.5814 4.65026 16.6222 4.67194 16.6442C4.69333 16.6659 4.73208 16.6878 4.80035 16.6878H13.1993C13.2675 16.6878 13.3063 16.6659 13.3277 16.6442C13.3494 16.6222 13.372 16.5814 13.372 16.5095V16.5035ZM4.38866 16.5095C4.38866 16.7599 4.55332 16.9267 4.80035 16.9267H13.1993C13.4463 16.9267 13.611 16.7599 13.611 16.5095L14.2694 3.57668H3.64756L4.38866 16.5095ZM6.35674 1.49059C6.35674 1.49059 6.35674 1.4906 6.35674 1.49059L6.35684 2.50331H11.6428V1.49059C11.6428 1.41876 11.6201 1.37792 11.5985 1.35596C11.5771 1.33427 11.5383 1.31235 11.47 1.31235H6.5295C6.46122 1.31235 6.42248 1.33427 6.40108 1.35596C6.3794 1.37792 6.35674 1.41878 6.35674 1.49059ZM6.5295 1.07341C6.28245 1.07341 6.1178 1.24027 6.1178 1.49059L6.11793 2.74225H11.8817V1.49059C11.8817 1.24025 11.7171 1.07341 11.47 1.07341H6.5295Z"
      fill="#211B28"
      fill-rule="evenodd"
    />
    <path
      d="M8.5875 5.66042H9.41086V14.8387H8.5875V5.66042Z"
      fill="#211B28"
    />
    <path
      clip-rule="evenodd"
      d="M8.34856 5.66042C8.34856 5.52845 8.45553 5.42148 8.5875 5.42148H9.41086C9.54282 5.42148 9.6498 5.52845 9.6498 5.66042V14.8387C9.6498 14.9707 9.54282 15.0776 9.41086 15.0776H8.5875C8.45553 15.0776 8.34856 14.9707 8.34856 14.8387V5.66042ZM8.5875 5.66042V14.8387H9.41086V5.66042H8.5875Z"
      fill="#211B28"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;
